mlflow/transformers/__init__.py

```diff
@@ -967,9 +970,8 @@ def log_model(  # noqa: D417
             model. The model signature can be inferred using `infer_signature` function
             of `mlflow.models.signature`.
 
-            Example:
-
             .. code-block:: python
+                caption: Example
```

Comment:
This might be a bug in ruff. Ruff thinks the arguments after `Example:` are not documented. As a workaround, replaced it with `caption`.